src/main/java/com/programmers/smrtstore/domain/category/application/CategoryService.java

```diff
@@ -0,0 +1,42 @@
+package com.programmers.smrtstore.domain.category.application;
+
+import com.programmers.smrtstore.domain.category.domain.entity.Category;
+import com.programmers.smrtstore.domain.category.exception.CategoryAlreadyExistException;
+import com.programmers.smrtstore.domain.category.exception.CategoryNotFoundException;
+import com.programmers.smrtstore.domain.category.infrastructure.CategoryJpaRepository;
+import com.programmers.smrtstore.domain.category.presentation.res.CategoryResponse;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.lang.NonNull;
+import org.springframework.stereotype.Service;
+import org.springframework.validation.annotation.Validated;
+
+@Validated
+@Service
+@RequiredArgsConstructor
+public class CategoryService {
+
+    private final CategoryJpaRepository categoryJpaRepository;
+
+    public CategoryResponse addCategory(@NonNull String value) {
```

Comment:
```
A common Spring annotation to declare that annotated elements cannot be null.
Leverages JSR-305 meta-annotations to indicate nullability in Java to common tools with JSR-305 support and used by Kotlin to infer nullability of Spring API.
Should be used at the parameter, return value, and field level. Method overrides should repeat parent @NonNull annotations unless they behave differently.
Use @NonNullApi (scope = parameters + return values) and/or @NonNullFields (scope = fields) to set the default behavior to non-nullable in order to avoid annotating your whole codebase with @NonNull.
Since:
5.0
See Also:
NonNullApi, NonNullFields, Nullable
Author:
Sebastien Deleuze, Juergen Hoeller
```
@NonNull 에 대한 설명입니다.